src/RenderWebGL.js

```diff
@@ -615,9 +621,30 @@ class RenderWebGL extends EventEmitter {
             return false;
         }
 
+        measure.start('newWay');
+
+        const point = twgl.v3.create(0, 0);
+        const iterator = bounds.points(point);
+        const drawable = this._allDrawables[drawableID];
+        const candidates = candidateIDs.map(id => this._allDrawables[id]);
+        let newWay = false;
+        while (iterator.next().value) {
+            if (drawable.isTouching(point)) {
+                if (candidates.some(d => d.isTouching(point))) {
+                    newWay = true;
+                    break;
+                }
+            }
+        }
+
+        measure.end('newWay');
+        measure.start('oldWay');
+
+        // return newWay;
+
         // Limit size of viewport to the bounds around the target Drawable,
         // and create the projection matrix for the draw.
-        gl.viewport(0, 0, bounds.width, bounds.height);
+        gl.viewport(0, 0, bounds.width + 1, bounds.height + 1);
```

Comment:
I was actually detecting a positive here that the old way didn't -- the old way doesn't detect collisions on the far right side of a cat whisker for instance (when the cat is asking...)